cfme/tests/automate/test_domain.py

```diff
@@ -182,8 +182,15 @@ def test_domain_name_wrong(appliance):
         initialEstimate: 1/60h
         tags: automate
     """
-    with pytest.raises(Exception, match='Name may contain only'):
-        appliance.collections.domains.create(name='with space')
+    wrong_domain = 'Dummy Domain'
```

Comment:
@ganeshhubale please add comment or steps in `doc-string`. as you took "Dummy Domain" other contributes  don't know that its not going to allow space.